met-etl/src/etl_project/services/ops/submission_etl_service.py

```diff
@@ -180,9 +177,12 @@ def _extract_submission(form_questions, met_survey, metsession, submission, mete
                 elif component_type == FormIoComponentType.CHECKBOX.value:
                     _save_checkbox(metetlsession, context, answer_key, component, etl_survey, user, submission,
                                    submission_new_runcycleid)
-                elif component_type == FormIoComponentType.TEXT.value:
-                    _save_text(metetlsession, context, etl_survey, component, answer_key, user, submission,
-                               submission_new_runcycleid)
+                elif component_type == FormIoComponentType.SELECTLIST.value:
+                    _save_select(metetlsession, context, answer_key, component, etl_survey, user, submission,
```

Comment:
can you do another PR just by changing metetlsession to met_etl_session ?

Reply:
Ok sure will do in another PR.